### PR TITLE
add translation for product_nature_variants

### PR DESCRIPTION
**Pull request by thib44**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -215,8 +215,6 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
-
-  gem 'letter_opener'
```
Comment (Aquaj) [inline Gemfile]: Why did you remove letter_opener ?

Comment (thib44) [reply Gemfile]: @Aquaj Done

File: config/locales/fra/access.yml [hunk 2]
```diff
@@ -104,4 +104,4 @@ fra:
       transports: "Livraisons"
       users: "Utilisateurs"
       wine_tanks: "Cuves à vin"
-      workers: "Travailleurs"
\ No newline at end of file
+      workers: "Travailleurs"
```
Comment (burisu) [inline config/locales/fra/access.yml]: Can you `git checkout master` changes if no real changes

File: config/locales/fra/action.yml [hunk 3]
```diff
@@ -1855,6 +1855,7 @@ fra:
     production_support: "Support de production"
     productions: "Productions"
     productions_map: "Carte des productions" #?
+    product_nature_variants: "Variantes de produits"
```
Comment (burisu) [inline config/locales/fra/action.yml]: English first please

**Comment by Aquaj** [comment]
Already fixed
